server/plugin.go

```diff
@@ -142,25 +144,33 @@ func (p *Plugin) updateOrAppend(originals []analyticsData, upsert analyticsData)
 	return append(originals, upsert)
 }
 
-func (p *Plugin) getChannelName(key string) (string, string, *model.AppError) {
+func (p *Plugin) getChannelName(key string) (string, string, string, *model.AppError) {
 	channel, err := p.API.GetChannel(key)
 	if err != nil {
-		return "", "", &model.AppError{
+		return "", "", "", &model.AppError{
 			Message:       "Can't retreive channel name",
 			DetailedError: err.Error(),
 		}
 	}
 	if channel.IsGroupOrDirect() {
-		return "Private", "Private", nil
+		return "", "Private", "", nil
```

Comment:
I'm 3/5 not calling this `Private` as this is reserved to private channels. Maybe just call it `DM`?

Reply:
:heavy_check_mark: